.github/workflows/build.yml

```diff
@@ -28,9 +28,9 @@ jobs:
       - name: Setup node and install deps
         uses: actions/setup-node@v2
         with:
-          node-version: '16.15.0'
-          check-latest: false
-      - run: npm --prefix frontend ci --no-optional
```

Comment:
Why is no-optional removed and not in addition to legacy-peer-deps? Are optional dependencies still ignored in the prod build?

Reply:
The prod build is another question, it should be based upon the package-lock I guess, as we were using the wrong flag the whole time I'm wondering how this did work at all.

Reply:
Most likely the `optional=false` setting in the `.npmrc` file prevented optional dependencies on the prod build. So there might still be a fallback in npm.
But this solution seems the way to go, approved :+1: